scripts/deploy/github.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+# Script for deploying GitHub bootstrap stack
+
+set -e  # Exit immediately if a command exits with a non-zero status
+
+# Function to print messages
+print_message() {
+    echo "===> $1"
+}
+
+# Function to setup and activate virtual environment
+setup_venv() {
+    print_message "Setting up virtual environment..."
+    python3.10 -m venv .venv
+    source .venv/bin/activate
+    pip install -r requirements.txt
+    pip install pip --upgrade
```

Comment:
wouldn't we want to upgrade pip before its used to install the requirements?

Reply:
nah you have to install it inside the venv for some reason - or at least that does work for me.